apps/docs/src/app/core/component-docs/toolbar/toolbar-docs.component.html

```diff
@@ -33,6 +33,43 @@
 <code-example [exampleFiles]="toolbarOverflowExample"></code-example>
 <separator></separator>
 
+<fd-docs-section-title [id]="'Toolbar-overflow-priority'" [componentName]="'toolbar'">
```

Comment:
unnecessary square bindings
```suggestion
<fd-docs-section-title id="Toolbar-overflow-priority" componentName="toolbar">
```